tests/rptest/remote_scripts/compute_storage.py

```diff
@@ -33,6 +74,14 @@ def safe_listdir(p: Path) -> list[Path]:
         return []
 
 
+def compute_size_for_file(file: Path):
+    if file.suffix == '.log':
+        reader = SegmentReader(io.BytesIO(file.read_bytes()))
```

Comment:
Wondering if it makes sense to have a fast path if last byte is non 0.

Reply:
I can give it a try.